rules/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector.php

```diff
@@ -186,36 +183,74 @@ private function isClassMethodCalledInLocalMethodCall(Class_ $class, string $cla
             }
 
             // the method is used
-            if ($this->isName($methodCall->name, $classMethodName)) {
+            if ($this->isName($call->name, $classMethodName)) {
                 return true;
             }
         }
 
         return false;
     }
```

Comment:
Could you extract this method to decoupled service? It seems we might re-use it.

Reply:
implemented https://github.com/rectorphp/rector-src/pull/598/commits/0c1972bbd1fb28c9221f1f67301e9d49b5871954